File: mario.c
#include <cs50.h>
#include <stdio.h>

int get_positive_int(string prompt);

int main(void)
{
    int height = get_positive_int("Height: ");
    printf("Stored: %i\n", height);
    for (int i = 0; i < height; i++)
    {
        for (int j = 1; j < height - i ; j++)
        {
            printf(" ");
        }
            for (int k = 0; k <= i; k++)
            {
                printf("#");
            }
        printf("\n");
    }     
}

int get_positive_int(string prompt)
{
    int n;
    do
    {
        n = get_int("%s", prompt);
    }
    while (n < 1 || n > 8);
    return n;
}

File: mario.py
# Problem Set 6, Sentimental-Mario(less), Python, 13 July 2019

from cs50 import get_int

# Prompt user for a positive height no more than 8
while True:
    n = get_int("Height: ")
    if n > 0 and n < 9:
        break

for i in range(n):
    print(" " * (n - i - 1), end="")
    print("#" * (i + 1), end="")
    print()